test/federated/test_federated_client.py

```diff
@@ -196,7 +196,7 @@ def forward(self, x):
             )
             loss_after = evaluate_model(fed_client, model_id, loss_fn, data, target)
 
-        assert loss_after < loss_before
+        assert loss_after <= loss_before
```

Comment:
This was added because ot this:
```
2020-05-11T19:38:50.7796829Z     
2020-05-11T19:38:50.7797216Z >           assert loss_after < loss_before
2020-05-11T19:38:50.7797611Z E           assert tensor(0.6931, grad_fn=<NllLossBackward>) < tensor(0.6931, grad_fn=<NllLossBackward>)
2020-05-11T19:38:50.7797963Z 
2020-05-11T19:38:50.7798354Z test/federated/test_federated_client.py:199: AssertionError
```